test/test_job.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from pytest import fixture
+import pytest
+import vcr
+
+from marquez_client.marquez import MarquezClient
+
+
+@fixture(scope='class')
+def set_valid_env():
+    os.environ[MarquezClient.MARQUEZ_HOST_KEY] = "localhost"
+    os.environ[MarquezClient.MARQUEZ_PORT_KEY] = "8080"
+
+
+@fixture(scope='class')
+def marquez_client(set_valid_env):
```

Comment:
hm, do you want a `class` scoped client? it is being mutated across tests

Reply:
Good catch. Will adjust to method scope.